src/SmartComponents/completedTaskDetailsHelpers.js

```diff
@@ -56,7 +56,8 @@ export const fetchTaskJobs = async (taskDetails, setError) => {
         if (
           item.status === 'Timeout' ||
           item.status === 'Failure' ||
-          item.results.alert === 'true'
+          item.results.alert === 'true' ||
```

Comment:
@mhuth I have checked PR locally and it works fine. But I see if alert param is returned - it's boolean, not string. Are there any cases when we get this as string?

Reply:
@LightOfHeaven1994 I don't think so, but I kept the code checking if its a string in there, just in case I'm wrong or missed something.